.travis.yml

```diff
@@ -24,12 +24,15 @@ addons:
 
 before_install:
   - .travis-scripts/install.sh
-
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then .travis-scripts/osx-opencv-install.sh; fi
 # Only do an assemble when we aren't building a pull request
 install:
   - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && ./gradlew :ui:assemble --stacktrace || ./gradlew --stacktrace '
 
 script:
+  - if [[ "TRAVIS_OS_NAME" == "osx" ]]
```

Comment:
Each line in `script` is a separate command. This should be a multiline string. See http://stackoverflow.com/questions/3790454/in-yaml-how-do-i-break-a-string-over-multiple-lines/21699210#21699210